### PR TITLE
Add KiamSTSIssuingErrors

### PR DESCRIPTION
**Pull request by njuettner**
This hopefully will block `ExternalDNSCantAccessRegistry` paging during the night, by adding STSIssue alert.

`ExternalDNSCantAccessRegistry` uses `cancel_if_kiam_has_errors`, see
 https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/external-dns.rules.yml#L26, which should be inhibited here https://github.com/giantswarm/prometheus-rules/blob/master/helm/prometheus-rules/templates/alerting-rules/inhibit.all.rules.yml#L32-L40.
### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml [hunk 1]
```diff
@@ -27,4 +27,21 @@ spec:
         severity: page
         team: firecracker
         topic: kiam
+  - name: kiam
+    rules:
+    - alert: KiamSTSIssuingErrors
+      annotations:
+        description: '{{`Kiam pod {{ $labels.namespace}}/{{ $labels.pod_name }} on {{ $labels.cluster_id}}/{{ $labels.cluster }} has increased STS issuing errors.`}}'
+        opsrecipe: kiam_sts_issuing_errors_total/
+      expr: increase(kiam_sts_issuing_errors_total[10m]) > 0
```
Comment (whites11) [inline helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml:36]: Maybe we could allow some errors? Like 5 or 10 dunno

**Review by whites11 — APPROVED** [review]
Otherwise LGTM

File: helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml (hunk 1 above)
Comment (njuettner) [inline helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml:36]:
```suggestion
      expr: increase(kiam_sts_issuing_errors_total[10m]) > 5
```

Comment (njuettner) [reply helm/prometheus-rules/templates/alerting-rules/kiam.rules.yml]: Makes sense 👍🏻

**Pull request merged**